vllm/worker/habana_model_runner.py

```diff
@@ -1467,6 +1467,11 @@ def warmup_model(self, kv_caches: List[torch.Tensor]) -> None:
                     len(self.decode_buckets),
                     list(sorted(self.decode_buckets)))
 
+        if not htorch.utils.internal.is_lazy() and not self.enforce_eager:
+            cache_size_limit = len(self.prompt_buckets) + len(self.decode_buckets)
+            torch._dynamo.config.cache_size_limit = cache_size_limit
+            torch._dynamo.config.accumulated_cache_size_limit = cache_size_limit * 8
```

Comment:
What is the origin of this "8"? 
Please add comment or calculate it from some other parameters

Reply:
I just followed the original ratio between the 2 parameters(8 for cache_size_limit and 64 for accumulated_cache_size_limit).